i18n/en.pot

```diff
@@ -112,8 +112,8 @@ msgstr "How to create new catchment areas"
 msgid "Click"
 msgstr "Click"
 
-msgid "Create"
-msgstr "Create"
+msgid "Create catchment areas"
+msgstr "Create catchment areas"
```

Comment:
Some places we say "catchment area" and others we say "catchment areas" -- I'm not sure which is correct but I think we should be consistent